src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,8 +48,7 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema ++
-      submissionAuditStatusQuery.schema ++
-      exprEvalQuery.schema
```

Comment:
This was just added for the tests-go-faster PR but I think it was a mistake to do so.